interface/orders/single_order_results.php

```diff
@@ -138,7 +144,8 @@
    f.submit();
   }
  }
- window.close();
+ let stayHere = './single_order_results.php?orderid=' + encodeURIComponent('<?php echo $orderid; ?>');
```

Comment:
there are 2 options here for html escaping:
````
let stayHere = './single_order_results.php?orderid=' + encodeURIComponent(<?php echo js_escape($orderid); ?>);
````
or
````
let stayHere = './single_order_results.php?orderid=' + <?php echo js_url($orderid); ?>);
````

Reply:
I'm not even going to argue but still say this doesn't need to be escaped.

Reply:
humor me and consider picking an option :)
(If we someday convert to uuid's and don't force intval on it, then it could turn into a vulnerability :) )
I'm also happy to do this after it gets into the codebase
(btw, the use of the js_escape() function has really worked out like a charm for the js escaping)